DataFormats/PatCandidates/src/classes_def_objects.xml

```diff
@@ -341,6 +339,21 @@
     <![CDATA[delete track_.exchange(nullptr);
     ]]>
   </ioread>
+  <ioread sourceClass="pat::PackedCandidate" source="uint16_t packedCovarianceDxyDxy_; uint16_t packedCovarianceDxyDz_; uint16_t packedCovarianceDzDz_;   int8_t packedCovarianceDlambdaDz_; int8_t packedCovarianceDphiDxy_;  int8_t packedCovarianceDptDpt_; int8_t packedCovarianceDetaDeta_;  int8_t packedCovarianceDphiDphi_ " version="[-27]" targetClass="pat::PackedCandidate"
```

Comment:
Sam's code is in 910pre2. This is production version for the purpose of changes like this.